### ISSUE TITLE
TOPICS: 'Read more' missing from preview

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1479]
When looking at the preview, it looks like there is no more text. But there is, so should be a 'read more'. It's a 'starting with discussion' topic. 

![image](https://github.com/user-attachments/assets/80bd95d9-b135-467f-8a18-1b40df2f8413)

**Comment by BeccaMelhuish** [comment]
I then went back to 1.Compose tab to copy the text in case it was lost on publishing, and when I came back to 4.Preview, 'Read more' was there: 

![image](https://github.com/user-attachments/assets/44ae0763-e4b5-45a2-af60-61a45a9ae2af)

**Comment by BeccaMelhuish** [comment]
This just happened to me again, FYI

**Comment by BeccaMelhuish** [comment]
@ssin1901 considering adding the 'high priority' label, but maybe you could check how much it is happening in testing, and see how high priority you think it is? Maybe it's for long topic texts only, not sure?

**Comment by BeccaMelhuish** [comment]
Happened to me again just now so will add high priority :) 
(It means it's not possible to check the rest of the topic text, so is quite important)

**Comment by ilmartyrk** [comment]
Has something to do with syncing the topic in the creating process. Should be quite an easy fix, but will take some time to find the proper solution

**Comment by BeccaMelhuish** [comment]
@ssin1901 as discussed will put it to QA to see how prevalent the issue is

**Comment by ssin1901** [comment]
@BeccaMelhuish @ilmartyrk might be wrong but seems to be linked to a larger syncing issue. 

With slower connections sometimes even the title and description disappear.

@ilmartyrk if you agree we could group them into a single issue. @BeccaMelhuish And I suggest moving it to Backlog - Soon, as Ilmar mentioned that it will take some time to find a proper solution.

**Comment by BeccaMelhuish** [comment]
@ssin1901 Yes I remember @ilmartyrk mentioned this was to do with syncing times. Seems to make sense to make a new issue with all the related issues in one I agree. And then close this one. 

Do you have an idea from the testing of how much this might be happening for users? As if fairly frequent (for me it has happened a good few times), then I think we should keep it quite high priority?

**Comment by ssin1901** [comment]
@BeccaMelhuish I made another issue about syncing. If Ilmar confirms that's the case for the other issue, I will add this one there.

**Comment by ssin1901** [comment]
@BeccaMelhuish I think this can be moved to in Progress right?

**Comment by BeccaMelhuish** [comment]
@ssin1901 Would it be possible to make a 'group issue' with checklist linking to all the syncing related issues you've found so they can be tackled all together? Including the authentication required one. Then I'll prioritise them all to be worked on together :)

**Comment by BeccaMelhuish** [comment]
Adding 'grouped' label and moving to 'no status' column as it is now covered in https://github.com/citizenos/citizenos-fe/issues/1788
